config/vufind/EDS.ini

```diff
@@ -403,3 +403,14 @@ AU = None
 ; the LibGuides URLs if the same name is present here.
 ;url[Expensive Science Database] = https://some.url
 ;url[Cool Humanities Database] = https://another.url
+
+; Define whether to display 'Short' or 'Long' author data on the EDS search result list page.
+; - 'Short' is fetched from the RecordInfo section of a record (the 
+; 'BibRecord/BibRelationships/HasContributorRelationships/*PersonEntity/Name/NameFull' data).
+; - 'Long' is fetched from the item 'Au' group data.
+; If you do set 'ResultListPage' to 'Short' then you can define how many author names to 
+; display in AuthorNameLimit. If a record has more author names than you define in AuthorNameLimit
+; then VuFind will display the more_authors_abbrev text.
+;[AuthorResultListInfo]
+;ResultListPage = 'Short'
```

Comment:
If you're going to use mixed-case settings like Short and Long, I'd suggest making the config processing logic case-insensitive to prevent people from getting confused by hard-to-notice configuration subtleties (like s vs. S).

Reply:
Thanks @demiankatz , I've added the config processing case-insensivity you suggested in the next review comment in my latest follow-up. So I think this is now addressed